packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -0,0 +1,62 @@
+Dropdown containers display a menu with additional items as soon as a user clicks on the trigger element, e.g. a dropdown button. Clicking on one of the items in the dropdown performs the action. We typically use dropdowns to allow users to select one option from a list of available choices.
+
+For information about the trigger element that opens the dropdown, please refer to our [dropdown button](buttons/dropdown-button.md) and [split button](buttons/split-button.md) components.
+
+![Overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2353-2278&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+1. Dropdown container
+2. Item
+3. Header
+4. Quick actions
+5. Submenu
+6. Checked
+7. Separator
+8. Icon
+9. Label
+
+## Options
+
+- **Header:** Add a header for the dropdown container. This allows users to better understand which elements are in the dropdown.
+- **Quick Actions:** Add a quick action bar. Add 3 to 5 actions/items to the quick actions. We typically use quick actions to access common functions such as cut, copy and paste.
+- **Checked:** Mark selected items in the dropdown with a check mark. We typically use check marks when an item can be activated (e.g. for the dropdown "Sort by:" selection: name, modified date, create date).
+- **Submenu:** Add a submenu for a multi-level dropdown. We typically use a submenu when there is a long item list and different systematic categorization within the items.
+- **Separator:** Add a separator to visually divide items from each other. We normally use a separator to isolate individual elements from a cohesive item list.
+- **Icon:** Icons can also be displayed with item labels. We typically use an icon in addition to the label when it has a recognizable benefit.
```

Comment:
What is a 'recognizable benefit'?

Reply:
Idea:
**Icon:** Icons can be displayed to support the label and make the item faster discoverable. The icon should be widely known for representing the action or function among your users .

Reply:
```suggestion
- **Icon:** Icons can be displayed to support the label and make the item more easy to discover by the user. The icon should be widely known for representing the action or function among your users.
```